pkg/operator/ceph/csi/spec.go

```
@@ -367,6 +367,44 @@ func startDrivers(namespace string, clientset kubernetes.Interface, ver *version
 			return errors.Wrap(err, "failed to load cephfs plugin service template")
 		}
 	}
+
+	pv, _ := clientset.CoreV1().PersistentVolumes().List(metav1.ListOptions{})
```

dont discard errors.

---

okay.